src/java/picard/analysis/directed/CollectTargetedPcrMetrics.java

```diff
@@ -17,15 +17,38 @@
  * more information
  */
 @CommandLineProgramProperties(
-        usage = "Calculates a set of metrics to Illumina Truseq Custom Amplicon sequencing from an aligned SAM" +
-                "or BAM file. If a reference sequence is provided, AT/GC dropout metrics will " +
-                "be calculated, and the PER_TARGET_COVERAGE option can be used to output GC and " +
-                "mean coverage information for every target.",
-        usageShort = "Produces Targeted PCR-related metrics given the provided SAM/BAM",
+        usage = CollectTargetedPcrMetrics.USAGE_SUMMARY + CollectTargetedPcrMetrics.USAGE_DETAILS,
+        usageShort = CollectTargetedPcrMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectTargetedPcrMetrics extends CollectTargetedMetrics<TargetedPcrMetrics, TargetedPcrMetricsCollector> {
-
+    static final String USAGE_SUMMARY = "Produces targeted PCR-related metrics for a given SAM/BAM file.  ";
+    static final String USAGE_DETAILS = "This tool calculates a set of metrics to TSCA sequencing from an aligned SAM or" +
+            "BAM file.  TruSeq Custom Amplicon (TSCA) is an Illumina kit that allows researchers to sequence targeted " +
+            "regions of interest in a genome.  Although there are many applications for using this targeted " +
+            "approach, it is frequently used for genomic regions of high or low GC content.  For additional information, please see:" +
+            "<br /><br />" +
+            "www.illumina.com/content/dam/illumina-marketing/documents/products/datasheets/datasheet_truseq_custom_amplicon.pdf." +
+            "<br /><br />" +
+            "If a reference sequence is provided, AT/GC dropout metrics will be calculated and the PER_TARGET_COVERAGE" +
```

Comment:
Specification of TruSeq is misleading here.  This metric is appropriate for multiple small-target technologies.